UI/window-basic-settings.cpp

```diff
@@ -2118,20 +2108,11 @@ void OBSBasicSettings::LoadAdvOutputStreamingSettings()
 	ui->advOutMultiTrack5->setChecked(audioMixes & (1 << 4));
 	ui->advOutMultiTrack6->setChecked(audioMixes & (1 << 5));
 
-	bool is_multitrack_output = false;
 	obs_service_t *service_obj = main->GetService();
 	const char *protocol = nullptr;
 	protocol = obs_service_get_protocol(service_obj);
 	if (protocol) {
-		is_multitrack_output = allowsMultiTrack(protocol);
-	}
-
-	if (is_multitrack_output) {
-		ui->advStreamTrackWidget->setCurrentWidget(
-			ui->streamMultiTracks);
-	} else {
-		ui->advStreamTrackWidget->setCurrentWidget(
-			ui->streamSingleTracks);
+		SwapMultiTrack(QT_UTF8(protocol));
```

Comment:
We call `QT_UTF8` here, but then call `QT_TO_UTF8` twice inside the `SwapMultiTrack` function. Why do we need to do this?

Reply:
The function is called twice in window-basic-settings-steam.cpp where protocol is a QString instead of a const char*. There were also multiple iterations while multi-tasking and moved some code around.
For the sake of simplicity and readability, i'm switching the arg to const char *